libs/components/src/next/properties/vl-properties.component.ts

```diff
@@ -0,0 +1,136 @@
+import { BaseLitElement, onChildListChange, webComponent } from '@domg-wc/common-utilities';
+import { globalStylesNext } from '@domg-wc/common-utilities/css/global-styles-decorator';
+import { CSSResult, html, PropertyDeclarations, PropertyValues, TemplateResult } from 'lit';
+import { buildProperties } from './vl-properties.builder';
+import propertiesStyles, { labelWidthRem } from './vl-properties.css';
+import { Column, Item, Props } from './vl-properties.model';
+import { vlElementsStyle } from '@domg-wc/elements';
+
+export const propertiesDefaults = {
+    labelWidth: 17,
+    props: [] as Props,
+} as const;
+
+@globalStylesNext()
+@webComponent('vl-properties-next')
+export class VlPropertiesComponent extends BaseLitElement {
+    private attributeProps: Props = propertiesDefaults.props;
+    private aggregatedProps: Props = propertiesDefaults.props;
+    private mutationObserverList: MutationObserver[] = [];
+    private labelWidth: number = propertiesDefaults.labelWidth;
+    private labelWidthSheet: CSSStyleSheet = new CSSStyleSheet();
+
+    static get styles(): (CSSResult | CSSResult[])[] {
+        return [vlElementsStyle, propertiesStyles];
+    }
+
+    static get properties(): PropertyDeclarations {
+        return {
+            props: { type: Array },
```

Comment:
Als je hier van `aggregatedProps` maakt, heb je `this.requestUpdate();` niet meer nodig in `observeLightPropertiesChange`

Reply:
Zoals besproken, verbetert dit niet per sé de werking van het component en zal deze suggestie niet doorgevoerd worden om geen neveneffecten te bekomen.